_replication-plugin/get-started.md

```diff
@@ -0,0 +1,279 @@
+---
+layout: default
+title: Get started
+nav_order: 10
+---
+
+# Get started with cross-cluster replication
+
+With cross-cluster replication, you index data to a leader index, and OpenSearch replicates that data to one or more read-only follower indices. All subsequent operations on the leader are replicated on the follower, such as creating, updating, or deleting documents.
+
+## Prerequisites
+
+Cross-cluster replication has the following prerequisites:
+- Both the leader and follower cluster must have the replication plugin installed.
+- If you've overridden `node.roles` in `opensearch.yml` on the remote cluster, make sure it also includes the `remote_cluster_client` role:
+
+   ```yaml
+   node.roles: [<other_roles>, remote_cluster_client]
+   ```
```

Comment:
Found some accidental indentation here.

Reply:
This is part of the bullet point above it so I wanted to keep it in line.